### DISCUSSION TITLE
Selfhosted storage getPublicUrl {"statusCode":"400","error":"ENOENT: no such file or directory

### DISCUSSION THREAD

**Discussion by 1270514689** [supabase/supabase#10327] [Questions] [answered]
Selfhosted storage getPublicUrl

bucket name：bucket
path：public/pic001.jpg

{"statusCode":"400","error":"ENOENT: no such file or directory, open '/var/lib/storage/stub/stub/bucket/public/pic001.jpg'","message":"ENOENT: no such file or directory, open '/var/lib/storage/stub/stub/bucket/public/pic001.jpg'"}

I used the default docker-compose.yaml  and the createSignedUrl is good.

**Comment by GaryAustin1** [comment]
Is the bucket set to public?   There is a public column in the storage buckets table.

**Reply by 1270514689** [reply]
yes，the bucket is public

**Comment by awalias** [comment, marked as answer]
duplicate of https://github.com/supabase/supabase/issues/10328 . Will close in favour of the issue 👍

**Reply by 1270514689** [reply]
ok
